analytics/analytics/docs/schemas/requests.py

```diff
@@ -0,0 +1,91 @@
+from uuid import UUID
+
+from pydantic import BaseModel, Field, validator
```

Comment:
Very cool! I've been wanting to use pydantic :rocket:

Reply:
Yeah, Pydantic is indeed very cool! It basically removed the need for most of the input validation inside the `on_get` and `on_post` handlers.